hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/ratis/utils/OzoneManagerRatisUtils.java

```diff
@@ -290,8 +289,6 @@ public static OMClientRequest createClientRequest(OMRequest omRequest,
       volumeName = keyArgs.getVolumeName();
       bucketName = keyArgs.getBucketName();
       break;
-    case EchoRPC:
```

Comment:
While we don't need this here we should probably add it to `OmUtils#isReadOnly`.

Reply:
Yes sure! it's in OmUtils#isReadOnly already! (https://github.com/apache/ozone/blob/master/hadoop-ozone/common/src/main/java/org/apache/hadoop/ozone/OmUtils.java#L273)

Reply:
Oh nice my bad I had an old version of master checked out locally.